src/main/kotlin/com/team4099/robot2022/config/constants/DrivetrainConstants.kt

```diff
@@ -0,0 +1,83 @@
+package com.team4099.robot2022.config.constants
+
+import com.team4099.lib.units.base.feet
+import com.team4099.lib.units.base.inMeters
+import com.team4099.lib.units.base.inches
+import com.team4099.lib.units.base.meters
+import com.team4099.lib.units.derived.degrees
+import com.team4099.lib.units.derived.div
+import com.team4099.lib.units.derived.radians
+import com.team4099.lib.units.derived.rotations
+import com.team4099.lib.units.derived.volts
+import com.team4099.lib.units.inMetersPerSecond
+import com.team4099.lib.units.perSecond
+import kotlin.math.sqrt
+
+object DrivetrainConstants {
+  const val TICKS = 4096
+
+  const val WHEEL_COUNT = 4
+  val WHEEL_DIAMETER = 4.inches
+  val DRIVETRAIN_LENGTH = 22.750.inches
+  val DRIVETRAIN_WIDTH = 22.750.inches
+
+  val DRIVE_SETPOINT_MAX = 15.feet.perSecond
+  val TURN_SETPOINT_MAX =
+      (DRIVE_SETPOINT_MAX.inMetersPerSecond / DRIVETRAIN_LENGTH.inMeters / 2 * sqrt(2.0)).degrees
+          .perSecond // 648
+
+  // cruise velocity in raw sensor units per 100 ms
+  val STEERING_VEL_NATIVE_MAX = 3.87.rotations.perSecond
```

Comment:
can we express this in degrees/sec? rotations/sec is less meaningful because we won't ever do a full rotation here